static/app/components/smartSearchBar/index.tsx

```diff
@@ -167,6 +167,10 @@ type Props = WithRouterProps & {
    * as the stream view where it is a top level concept
    */
   excludeEnvironment?: boolean;
+  /**
+   * An array of query fields to exclude from the search bar
+   */
+  excludeQueryField?: string[];
```

Comment:
hmm not sure how good it would be to add another prop here. The smartSearchBar is already quite bloated and if anything we'd be looking to remove props. Is there a way we could achieve this exclusion functionality by just removing `issue` from the `supportedTags` prop?

Reply:
Hmm, that makes sense to me, but can I just exclude using supportedTags? I'm assuming I would have to relist all the tags minus `issue`

Reply:
Assuming that you cannot exclude tags, does it makes sense to remove `excludeEnviornment` and replace that with `excludeTags`, this would make it a bit more generic and fullfill my usecase.

Reply:
yea I think that also works fine but that may require some additional refactoring for all the usages of `excludeEnvironment` 

and on the other hand for the `supportedTags` use case you are right you would probably have to take the list and filter out `issues`. I imagine it might require adding a new prop to the component to modify this https://github.com/getsentry/sentry/blob/cfe1c73672eafb0ef933ca14661fe7dd440182c6/static/app/components/events/searchBar.tsx#L216

Reply:
I think either approach is fine so if its easier to combine the `excludeEnvironment` with your use case into an `excludeTags` prop I'm good with that

Reply:
Sweet, I'm willing to combine in and do the appropriate refactor.
I'm thinking of doing this in two PRs, 
1. Add excludeTags to this PR
2. Remove excludeEnvironment and refactor codebase accordingly

Reply:
sounds good to me, thanks for taking on the refactor too